.github/workflows/nightly-coraza-check.yaml

```diff
@@ -0,0 +1,62 @@
+name: Nightly Coraza Check
+on:
+  pull_request:
+  schedule:
+    - cron: "0 4 * * *"
+
+env:
+  GO_VERSION: 1.18
+  TINYGO_VERSION: 0.25.0
+  WABT_VERSION: 1.0.29
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+
+      - name: Install Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: ${{ env.GO_VERSION }}
+          cache: true
+
+      - name: Install TinyGo and wabt
+        run: |
+          gh release download v${TINYGO_VERSION} -p '*.linux-amd64.tar.gz' -D ~ -R github.com/tinygo-org/tinygo
+          tar -xf ~/tinygo${TINYGO_VERSION}.linux-amd64.tar.gz  -C $HOME
+          echo "$HOME/tinygo/bin" >> $GITHUB_PATH
+
+          gh release download ${WABT_VERSION} -p '*-ubuntu.tar.gz' -D ~ -R github.com/WebAssembly/wabt
+          tar -xf ~/wabt-${WABT_VERSION}-ubuntu.tar.gz  -C $HOME
+          echo "$HOME/wabt-${WABT_VERSION}/bin" >> $GITHUB_PATH
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Cache TinyGo build
+        uses: actions/cache@v3
+        with:
+          path: |
+            ~/.cache/tinygo
+          key: ${{ runner.os }}-tinygo-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-tinygo-
+
+      - name: Get last commit of coraza
+        uses: sergeysova/jq-action@v2
```

Comment:
It's good to avoid unofficial actions where possible - I think the `gh` CLI can do this easily and is preinstalled

Reply:
I tried to find a way using something like `gh repo view https://github.com/corazawaf/coraza -b v3/dev --json` with no success. Any hunch?

Reply:
`gh` CLI provides access to the entire API with built in jq :)

`gh api repos/corazawaf/coraza/commits/v3/dev -q .sha`

Reply:
🙀